gui/UI.py

```diff
@@ -0,0 +1,127 @@
+                )
+            else:
+                priceLabel = Label(self.frame, text=self.price_vals[row], bg="#1f1f1f", fg="#e6b800").grid(
+                    column=0, row=2 + row, sticky="w", padx=5
+                )
+                avgTimeLabel = Label(self.frame, text=avg_time_text, bg="#1f1f1f", fg="#e6b800").grid(
+                    column=2, row=2 + row, sticky="w", padx=5
+                )
+
+        footerbgLabel = Label(self.frame, bg="#0d0d0d").grid(column=0, row=rows_used + 3, columnspan=3, sticky="w" + "e")
+
+        minPriceLabel = Label(self.frame, text="Low: " + str(self.price[0]), bg="#0d0d0d", fg="#e6b800")
+        minPriceLabel.grid(column=0, row=rows_used + 3, padx=10)
+
+        avgPriceLabel = Label(self.frame, text="Avg: " + str(self.price[1]), bg="#0d0d0d", fg="#e6b800")
+        avgPriceLabel.grid(column=1, row=rows_used + 3, padx=10)
+
+        maxPriceLabel = Label(self.frame, text="High: " + str(self.price[2]), bg="#0d0d0d", fg="#e6b800")
+        maxPriceLabel.grid(column=2, row=rows_used + 3, padx=10)
+
+        extrabgLabel = None
+        extrabgLabel2 = None
+        notEnoughLabel = None
+        manualSearchLabel = None
+
+        if self.not_enough:
+            extrabgLabel = Label(self.frame, bg="#0d0d0d")
+            extrabgLabel.grid(column=0, row=rows_used + 4, columnspan=3, sticky="w" + "e")
+            notEnoughText = "Found limited search results"
+            notEnoughLabel = Label(self.frame, text=notEnoughText, bg="#0d0d0d", fg="#e6b800")
+            notEnoughLabel.grid(column=0, row=rows_used + 4, columnspan=3)
+            extrabgLabel2 = Label(self.frame, bg="#0d0d0d")
+            extrabgLabel2.grid(column=0, row=rows_used + 5, columnspan=3, sticky="w" + "e")
+            manualSearchText = "Use alt+t to search manually"
+            manualSearchLabel = Label(self.frame, text=manualSearchText, bg="#0d0d0d", fg="#e6b800")
+            manualSearchLabel.grid(column=0, row=rows_used + 5, columnspan=3)
+
+class NoResult(GuiComponent):
+    def __init__(self):
+        super().__init__()
+        self.have_timeout = True
+    def add_components(self):
+        """
+        Assemble a simple informative window which tells the user
+        that we were unable to confidently price the current clipboard
+        item.
+        """
+
+        # Setting up Master Frame, only currently used for background color due to grid format.
+        masterFrame = Frame(self.frame, bg="#1f1f1f")
+        masterFrame.place(relwidth=1, relheight=1)
+
+        headerLabel = Label(self.frame, text="Not Enough Data", bg="#0d0d0d", fg="#e6b800")
+        headerLabel.grid(column=0, row=1, padx=5)
+
+        displayText = "Could not find enough data to confidently price this item."
+        annotation = Label(self.frame, text=displayText, bg="#0d0d0d", fg="#e6b800")
+        annotation.grid(column=0, row=2)
+
+priceInfo = PriceInfo()
+priceInfo.prepare_window()
```

Comment:
This should definitely not be here. It is being executed even when GUI is disabled (and also why are you creating window in advance? I thought whole point of this PR was to not do that)

Reply:
I added in if USE_GUI: 
The point of the PR was to fix ctrl+c and have a base GUI class to create windows from
they don't have to be there, but I don't know where else to put them, I thought it would be nice to have the UI stuff in the same place

Reply:
well the ctrl+c issue was fixed already and its in master. But ayway, my point is, the creation of objects is fine but why are you calling also prepare_window? that just do not makes sense to me, isnt it called when calling show at cursor method anyway?

Reply:
I changed it to use show/hide instead of recreating the window, but I can change it back.
In the end I don't think it matters

Reply:
Well if you are not doing create/destroy anymore then this whole thing can be done in a lot simpler manner, as you can simply extract the show_price from current master to separate component and just pass Gui object to its constructor and modify it. If you are showing/hiding the windows and never destroying them, then there is no point in even having multiple hidden windows instead of just updating one with new content like it is currently being done on master

Reply:
Its changed back, im working on adding a mod search option now